src/libraries/Microsoft.PowerFx.Core/Lexer/TexlLexer.cs

```diff
@@ -1567,7 +1572,11 @@ private Token LexStringLit()
                         _sb.Append(ch);
                         NextChar();
                     }
-                    else if (!CharacterUtils.IsFormatCh(ch))
+                    else if (_pfxV1)
+                    {
+                        _sb.Append(ch);
+                    }
+                    else if (!CharacterUtils.IsFormatCh(ch))

```

Comment:
shouldn't this be grouped together in one block?
`else if (_pfxV1 || !CharacterUtils.IsFormatCh(ch))
                    {
                        _sb.Append(ch);
                    }`

Reply:
Thanks for the suggestion.